### DISCUSSION TITLE
Pasting doesn't work in "Files'" address field

### DISCUSSION THREAD

**Discussion by FunkySalamander** [elementary/files#2173] [Q&A]
I'm on horus with the following non-default repos enabled:
-Jammy proposed
-Elementary daily & staging

When I open the 'Files' app & try to paste a path into the address field, it duplicates the '/home/username' part of the pasted text & fails to replace the highlighted text. Pasting the same clipboard entry elsewhere reveals the clipboard entry is indeed correct without the additional prepended text.

See attached screenshot.

![2023-04-25_File_Manager_Address_Bar_Issue](https://user-images.githubusercontent.com/125952784/234390864-afb229e4-c117-40c1-b670-6b025598a945.png)

**Comment by danirabbit** [comment]
Hey, thanks for reporting this. In the future please use the "Issues" tab to report issues. Thanks!

Moved here https://github.com/elementary/files/issues/2174
